openapi.yaml

```diff
@@ -300,6 +303,49 @@ paths:
       - events
 # {% if False %}
       - default
+# {% endif %}
+      x-codegen-request-body-name: body
+      x-openapi-router-controller: athenian.api.controllers.events_controller
+  /events/deployment/{name}/labels:
+    post:
```

Comment:
Let's opt for a CRUD here indeed. It should be cleaner.

Reply:
So is it ok if we add those two ?
- `GET` get the list of labels
- `PUT` set the list of labels

Reply:
It seems that we need `PATCH` instead of `PUT`. It will be easier for the user to request `append or replace label X=V` instead of rewriting everything. For example,
```
"upsert": {"X": "V"}, "delete": ["X"]
```